### PR TITLE
Makefile tweaks - xdg-open for ui on Linux; check for terraform binary in Makefile

### PR DESCRIPTION
**Pull request by jimmidyson**

### CODE REVIEW

File: options.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (pires) [inline options.json]: Why is this file here?

Comment (jimmidyson) [reply options.json]: Mistakenly added,,, removed now.

**Comment by pires** [comment]
Yay, more @jimmidyson love around this place!

**Comment by rimusz** [comment]
hehe, the starting ground for the new hires starts in this repo :)
